packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -70,14 +72,47 @@ export function Comments( {
 		setIsConfirmDialogOpen( false );
 	};
 
-	const blockCommentId = useSelect( ( select ) => {
+	useSelect( ( select ) => {
 		const clientID = select( blockEditorStore ).getSelectedBlockClientId();
-		return (
+		const clientBlocks = select( blockEditorStore ).getBlocks();
+		setBlocksList( clientBlocks );
```

Comment:
Please don't set sate inside `useSelect`, return the value instead

Reply:
@ellatrix - now the value has been returned inside `useSelect`